pom.xml

```diff
@@ -189,7 +189,7 @@
 		<!--TODO: Move transport related dependencies here pending https://jira.spring.io/browse/XD-3337  -->
 		<dependency>
 			<groupId>org.springframework.cloud</groupId>
-			<artifactId>spring-cloud-stream-binder-redis</artifactId>
+			<artifactId>spring-cloud-stream-binder-rabbit</artifactId>
```

Comment:
Also note that line 196 still has redis-starter.
I'm merging w/o the switch to rabbit, which deserves its own PR/commit I believe